internal/pkg/jsonnet/placeholder.go

```diff
@@ -0,0 +1,177 @@
+package jsonnet
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-jsonnet/ast"
+	"github.com/umisama/go-regexpcache"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/third_party/jsonnet/lib/parser"
+)
+
+const (
+	placeholderStart = "<<~~func:"
+	placeholderEnd   = "~~>>"
+	configIdFunc     = "ConfigId"
+	configRowIdFunc  = "ConfigRowId"
+)
+
+// nolint: gochecknoglobals
+var placeholderRegexp = regexpcache.MustCompile(fmt.Sprintf("%s([^:]+):([^~]+)%s",
+	regexp.QuoteMeta(placeholderStart),
+	regexp.QuoteMeta(placeholderEnd),
+))
+
+// ConfigIdPlaceholder generates <<~~func:ConfigId:["<ID>"]~~>>.
+func ConfigIdPlaceholder(id string) string {
+	return functionCallPlaceholder(configIdFunc, id)
+}
```

Comment:
Generovanie JsonNet prebieha takto:
- interne mame objekty ulozene ako structs, a orderedmaps.
- to nevieme priamo previest do `JsonNet`, bez implementovania vela kodu.
- ale vieme to previest do `Json` kódu, ktorý je zaroven aj validny `JsonNet`.
- nakoniec `Json` kof naformatujeme ako `JsonNet`.

Z tohto dovodu, ze to ide cez `Json`, nemozem do `JsonNet` kodu pridat jednoducho volanie funkcie `ConfigId`, pretoze to `Json` neumoznuje.

Preto, ako medzi-krok pouzijeme `placeholder`.
Vsade kde bude v nejakom stringu placeholder, napr. `<<~~func:ConfigId:["<ID>"]~~>>` ... sa tento fragment nahradi pri ukladani `JsonNet` za `ConfigId("<ID>")`.

Takto sa vyhneme koliziam, a vysledny kod je lepsie citatelny.